### DISCUSSION TITLE
Meaning of parameters in yaml files

### DISCUSSION THREAD

**Discussion by baby-coda** [damask-multiphysics/DAMASK#29] [Q&A] [answered]
Could you point me to where I can find the documentation for the different parameters in yaml files 
(what exactly N_sl, a_sl, h_0_sl-sl...etc corespond to ?)

**Comment by sharanroongta** [comment, marked as answer]
Hi, this damask overview paper has all the constitutive models of plasticity, and the parameters name are taken "more or less" from here: https://www.sciencedirect.com/science/article/pii/S0927025618302714?via%3Dihub
